projects/Benchmarks/ConsumerDispatching/ConsumerDispatcher.cs

```diff
@@ -35,7 +35,7 @@ public class BasicDeliverConsumerDispatching : ConsumerDispatcherBase
         public int Count { get; set; }
 
         [Params(1, 2)]
-        public int Concurrency { get; set; }
+        public ushort Concurrency { get; set; }
```

Comment:
Being CLS compliant is not something to aim for I guess?

Reply:
https://github.com/rabbitmq/rabbitmq-dotnet-client/issues/708 has been open for four years without comment, so my guess is "no".

In this case, I changed to `ushort` to restrict the acceptable range of input.

Reply:
I don't

I just feel that the some could also have been achieved with something like https://learn.microsoft.com/en-us/dotnet/api/system.argumentoutofrangeexception.throwifnegativeorzero?view=net-8.0#system-argumentoutofrangeexception-throwifnegativeorzero-1(-0-system-string) but I know this will trigger all the DDD people 😂

Reply:
The range check with exceptions is some quite common in the Azure SDK (not implying that is better, just stating it is being used there)

Reply:
Good to know! Those checks aren't available in `netstandard2.0` and I don't feel like copy-pasting code. I'll merge this soon, thanks @danielmarbach. I hope this helps out with your projects.